src/stru_multi/4C_stru_multi_microstatic.hpp

```diff
@@ -133,12 +133,23 @@ namespace MultiScale
     void output(Core::IO::DiscretizationWriter& output, const double time, const int istep,
         const double dt);
 
+    /*!
+     * @brief Write runtime output (displacement, tangent-stiffness, material IDs)
+     *
+     * Controlled via the RUNTIMEOUTPUT_GP flag, this function allows runtime output for
```

Comment:
Mentioning a specific input parameter is not very helpful, IMO. Why not refer to the relevant parameters in the code, e.g. the constructor arguments?